test/acceptance/tools/http_utils.py

```diff
@@ -0,0 +1,142 @@
+UNAUTHORIZED           = u'unauthorized'
+NOT_FOUND              = u'Not Found'
+BAD_METHOD             = u'Bad Method'
+NOT_ACCEPTABLE         = u'Not Acceptable'
+CONFLICT               = u'Conflict'
+UNSUPPORTED_MEDIA_TYPE = u'Unsupported Media Type'
+INTERNAL_SERVER_ERROR  = u'Internal Server Error'
+
+status_codes = {OK: 200,
+                CREATED: 201,
+                NO_CONTENT: 204,
+                MOVED_PERMANENTLY: 301,
+                REDIRECT: 307,
+                BAD_REQUEST: 400,
+                UNAUTHORIZED: 401,
+                NOT_FOUND: 404,
+                BAD_METHOD: 405,
+                NOT_ACCEPTABLE:406,
+                CONFLICT: 409,
+                UNSUPPORTED_MEDIA_TYPE: 415,
+                INTERNAL_SERVER_ERROR: 500}
+
+def print_request(method, url, headers, body):
+    """
+    Show Request in console
+    :param method: method used
+    :param url: url used
+    :param headers: header used
+    :param body: body used
+    """
+    print "------------------------------ Request ----------------------------------------------"
+    print "url: " + str(method) + "  " + str(url)+"\n"
+    if headers is not None:
+        print "Header: " + str(headers) + "\n"
+    if body != EMPTY:
+        print "Body: "  + str(body) + "\n\n\n"
+    print "----------------------------- End request ---------------------------------------------\n\n\n\n"
+
+def print_response(response):
+    """
+    Show response in console
+    :param response: http code, header and body returned
+    """
+    body = response.text
+    headers = response.headers
+    print "---------------------------------- Response ----------------------------------------------"
+    print "status code: " + str(response.status_code) + "\n"
+    if headers is not None:
+        print "Header: " + str(headers) + "\n"
+    if body != EMPTY:
+        print "Body: " + str(body) + "\n\n\n"
+    print "--------------------------------- End Response --------------------------------------------"
+def request (method, **kwargs):
+    """
+    launch a request
+    :param method: POST, GET, PUT, DELETE methods (MANDATORY)
+    :param url:  endpoint (with port and path)
+    :param headers: headers in request
+    :param data: payload if is required
+    :param Param: queries parameter if are required
+    :param allow_redirects: if redirect id allowed
+    :param verify: if the SSL is verified
+    :return: response (code, headers and body)
+
+    Note: two lines are comments because these are only used to internal debug in tests, 
+	      the first show the request and the second, show the response
+    """
+    try:
+        Url = kwargs.get(URL, EMPTY)
+        Headers = kwargs.get(HEADERS, None)
+        Body = kwargs.get (DATA, EMPTY)
+        Parameters = kwargs.get (PARAM, EMPTY)
+        Redirect = kwargs.get(ALLOW_REDIRECTS, True)
+        VerifySSL = kwargs.get(VERIFY, False)
+        #print_request(method, Url, Headers, Body)
+        if method == POST:
```

Comment:
The request library has a request method, where the method can be set dinamicaly, like 
```
requests.request(method.lower(),.......)
```

Its only a comment to no repeat each sentence changing only the method